app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class='recipe_info'>
+<h1 class= 'recipe_name_list'><%=@recipe.name%></h1>
+
+<div class='recipe_pre_pub'>
+<h2>Preparation_time: <%=@recipe.preparation_time%></h2>
```

Comment:
Kindly add authorization on the public recipe details to make sure that only the owner can modify the recipe details.

![image](https://github.com/Fene-87/recipe_app/assets/38088818/14920ad3-f247-4d69-a3bc-e2e2e8edb07b)

- Currently, I am able to add ingredients to recipes from another user. There should be authorization to prevent that from happening.
- Currently, I am able to modify or delete ingredients from the ingredients list. There should be authorization to prevent that from happening.